lib/Renard/Curie/Component/PageDrawingArea.pm

```diff
@@ -312,9 +312,9 @@ fun on_scroll_event_cb($window, $event, $self) {
 	return 0;
 }
 
-=func increment_scroll
+=fun increment_scroll
```

Comment:
This should stay as `=func`. This is part of the POD directives for the documentation.